pom.xml

```
@@ -0,0 +1,64 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+	<groupId>com.github.karllevik</groupId>
+	<artifactId>qmorph</artifactId>
+	<version>1.0</version>
+	<name>Q-Morph</name>
+	<build>
```

```suggestion
        <profiles>
                <profile>
                        <id>disable-java8-doclint</id>
                        <activation>
                                <jdk>[1.8,)</jdk>
                        </activation>
                        <properties>
                                <doclint>none</doclint>
                        </properties>
                </profile>
        </profiles>
	<build>
```
I found that I had to add the above for the javadoc to build without errors.

---

i'll add disable doclint to the plugin's default config

---

Actually, with `<failOnError>false</failOnError>` set, javadoc errors are merely reported by the linter, but will not stop the build.

---

Yes, but it still produces a lot of errors and warnings which clogs up the terminal. I guess ideally these errors and warnings should be addressed rather than suppressed/hidden.

But I really appreciate the work you've done, so I won't let this prevent me from approving/merging, and I can try to fix this minor issue myself later.